modules/test/playwright/pages/commerce/commerce-checkout-web/checkoutPage.ts

```diff
@@ -75,6 +89,10 @@ export class CheckoutPage {
 			name: 'Configuration',
 		});
 		this.countryInput = page.getByTitle('Country');
+		this.headingDeliveryGroupModal = (name: string) => {
+			return page.getByRole('heading', {exact: true, name});
+		};
+		this.iframeOkButton = page.getByLabel('close', {exact: true});
```

Comment:
why we are calling it iframe**Ok**Button if the label is "close"? can we call it iframeCloseButton?